client/index.html

```diff
@@ -287,8 +287,8 @@ <h1 class="title">Settings</h1>
 
 							<div class="col-sm-12">
 								<label class="opt">
-									<label for="highlights" class="sr-only">Custom highlights (comma-separated keywords)</label>
-									<input type="text" id="highlights" name="highlights" class="input" placeholder="Custom highlights (comma-separated keywords)">
+									<label for="highlights" class="sr-only">Custom highlights (comma-separated keywords, case-insensitive)</label>
```

Comment:
Not convinced about adding this to the label/placeholder, tbh. I think this would be expected as the default.

I guess the other option would be to put a checkbox for case sensitivity. But either way, I wouldn't put this in the label/placeholder.